src/main/swift/JIntellitype/JIntellitype.swift

```diff
@@ -0,0 +1,161 @@
+import Foundation
+import HotKey
+import AppKit
+
+var hotkeys: [jint: HotKey] = [:]
+var jvm: UnsafeMutablePointer<JavaVM?>? = nil
+var staticJobj: jobject? = nil
+var methodID: jmethodID? = nil
+
+@_silgen_name("Java_com_melloware_jintellitype_JIntellitype_initializeLibrary")
+public func initializeLibrary(env: UnsafeMutablePointer<JNIEnv?>, jobj: jobject) {
+    let jni = env.pointee!.pointee
+    let jvmPointer = UnsafeMutablePointer<UnsafeMutablePointer<JavaVM?>?>.allocate(capacity: 1)
+    let _ = jni.GetJavaVM(env, jvmPointer)
```

Comment:
Is there a reason to have the let assignment here even though it's not used? Couldn't you do the method call without the assignment? Same for the other occurrences of this.

Reply:
I added these only to "explicitly ignore" the values (otherwise Xcode was displaying warnings). :grin: As discussed, we can keep them for now.